### DISCUSSION TITLE
Wake vertical deflection for curl model

### DISCUSSION THREAD

**Discussion by Marcodep23** [NatLabRockies/floris#1049] [Q&A] [answered]
Hi everyone,

I would like to ask if it is possible to simulate tilted turbines with curl model. When I attempted to do so, I encountered the following error:
![image](https://github.com/user-attachments/assets/d8b7d6c5-bc61-42bf-967f-f9eb50c76765)
However, I noticed that this approach has been used in the following paper: https://www.nrel.gov/docs/fy19osti/73125.pdf.

Do you have any tips or suggestions for resolving this issue and making the simulation work?

Thank you in advance for your help!

Best regards,
Marco

**Comment by misi9170** [comment, marked as answer]
Hi Marco,

Thank you for your question. In the current version of FLORIS, only the Empirical Gaussian wake model supports vertical wake deflections due to tilt, so we'd encourage you to use that model if you are particularly interested in vertical deflections. 

It is possible that future FLORIS versions will reintroduce a tilt deflection capability within other wake models; however, this is not in our current plans. If you would like to see about making your own version of this work, you could take a look at how the it is implemented in the Empirical Gaussian model [here](https://github.com/NREL/floris/blob/main/floris/core/wake_deflection/empirical_gauss.py) and see if you can replicate the effect in other deflection models such as the gauss deflection model. If you decide to go down that route, we'd be interested to hear how you get on!

Best,
Misha
